tests/test_prettytable.py

```diff
@@ -2354,6 +2354,69 @@ def test_max_table_width_wide2(self) -> None:
 +---+-----------------+---+-----------------+---+-----------------+""".strip()
         )
 
+    def test_table_width_ctor1(self) -> None:
```

Comment:
What does "ctor" mean? Can we use something more descriptive?

Reply:
Sure, what about `test_table_width_on_init_wo_columns()`